src/index.html

```diff
@@ -3,11 +3,555 @@
+                class="compare__bike grid__item grid__item--ipad--2-5 grid__item--desktop--1-4"
+                data-aos="slide-right"
+                data-aos-duration="500"
+              >
+                <div class="bike">
+                  <div class="bike__photo-container">
+                    <img
+                      src="images/bikes/sporty.jpg"
+                      alt="bike 1"
+                      class="bike__photo"
+                    >
+                  </div>
+
+                  <h3 class="bike__info">
+                    Sporty 4
+                  </h3>
+
+                  <div class="bike__description">
+                    The iconic frame brought to a new performance height as a sporty, active ride.
+                  </div>
+
+                  <div class="bike__info bike__info--rotate">
+                    $ 2 590
+                  </div>
+                </div>
+              </div>
+
+              <div
+                class="compare__bike grid__item grid__item--ipad--2-5 grid__item--desktop--5-8"
+                data-aos="slide-up"
+                data-aos-duration="500"
+              >
+                <div class="bike">
+                  <div class="bike__photo-container">
+                    <img
+                      src="images/bikes/ride.jpg"
+                      alt="bike 2"
+                      class="bike__photo"
+                    >
+                  </div>
+
+                  <h3 class="bike__info">
+                    Ride in town ST
+                  </h3>
+
+                  <div class="bike__description">
+                    An open frame for an upright riding position as the most comfortable ride in town.
+                  </div>
+
+                  <div class="bike__info  bike__info--rotate">
+                    $ 2 590
+                  </div>
+                </div>
+              </div>
+
+              <div
+                class="compare__bike grid__item grid__item--ipad--2-5 grid__item--desktop--9-12"
+                data-aos="slide-left"
+                data-aos-duration="500"
+              >
+                <div class="bike">
+                  <div class="bike__photo-container">
+                    <img
+                      src="images/bikes/agile.jpg"
+                      alt="bike 3"
+                      class="bike__photo"
+                    >
+                  </div>
+
+                  <h3 class="bike__info">
+                    Agile ride 3
+                  </h3>
+
+                  <div class="bike__description">
+                    The lightweight frame that has earned its street tread as a sleek, agile ride.
+                  </div>
+
+                  <div class="bike__info  bike__info--rotate">
+                    $ 2 090
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </section>
+
+        <section class="page__section details" id="details">
```

Comment:
maybe bike-details ?

Reply:
bike-details already exist, it's separated class, there is also 'bike-details__title' with different parameteres than 'details__title'.